test/spec/controllers/document-import.js

```diff
@@ -272,6 +273,11 @@ describe('Controller: DocumentImportCtrl', function () {
           DocumentImportCtrl.importPastedContent(docImport.raw);
           expect(scope.yamlDocument.loadFailure).toBe(true);
         });
+
+        it('sets yamlDocument.loadFailure true', function () {
+          DocumentImportCtrl.importPastedContent(docImport.raw);
+          expect(scope.setLoading).toHaveBeenCalledWith(false);
```

Comment:
is ```setLoading``` already tested somewhere? I was thrown off by the test description, but assume that it's setting ```loadFailure``` inside that function call